hypervisor/include/arch/x86/trusty.h

```diff
@@ -121,4 +127,19 @@ void destroy_secure_world(struct vm *vm);
 
 void trusty_set_dseed(void *dseed, uint8_t seed_num);
 
+/**
+ * @brief Create Secure World EPT hierarchy
+ *
+ * Create Secure World EPT hierarchy, construct new PML4/PDPT, reuse PD/PT parse from
+ * vm->arch_vm->ept
+ *
+ * @param vm pointer to a VM with 2 Worlds
+ * @param gpa_orig original gpa allocated from vSBL
+ * @param size LK size (16M by default)
+ * @param gpa_rebased gpa rebased to offset xxx (511G_OFFSET)
+ *
+ */
+static void create_secure_world_ept(struct vm *vm, uint64_t gpa_orig,
```

Comment:
@YadongQi @dongyaozu @jren1 @anthonyzxu , please confirm it's OK to put this function declaration in the header file. This is needed in order to be able to add doxygen comments to it and be able to import it directly in the Trusty doc that this PR is creating.

Please review carefully the parameters and their definition as the document I used as the basis was outdated in that regard.